Settings.cpp

```diff
@@ -398,6 +398,11 @@ SoapySDR::RangeList SoapyRTLSDR::getFrequencyRange(
         const std::string &name) const
 {
     SoapySDR::RangeList results;
+    char manufact[256];
+    char product[256];
```

Comment:
There is a chance that `rtlsdr_get_usb_strings()` returns without setting the memory, both arrays need to be initialized (`= {0}`) for the `strcmp()` I'd say.

Reply:
Thanks, just fixed that.